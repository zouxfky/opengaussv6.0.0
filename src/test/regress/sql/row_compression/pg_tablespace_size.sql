CREATE TABLESPACE normal_tablespace RELATIVE LOCATION 'normal_tablespace';
SELECT pg_tablespace_size('normal_tablespace');
CREATE TABLE normal_table(id int) TABLESPACE normal_tablespace;
SELECT pg_tablespace_size('normal_tablespace');

CREATE TABLESPACE compress_tablespace RELATIVE LOCATION 'compress_tablespace';
SELECT pg_tablespace_size('compress_tablespace');
CREATE TABLE compressed_table_1024(id int) WITH(compresstype=2, compress_chunk_size=1024) TABLESPACE compress_tablespace;
SELECT pg_tablespace_size('compress_tablespace');
DROP TABLE normal_table;
DROP TABLESPACE normal_tablespace;
DROP TABLE compressed_table_1024;
DROP TABLESPACE compress_tablespace;

