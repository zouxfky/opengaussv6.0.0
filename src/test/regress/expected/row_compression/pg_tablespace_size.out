CREATE TABLESPACE normal_tablespace RELATIVE LOCATION 'normal_tablespace';
SELECT pg_tablespace_size('normal_tablespace');
 pg_tablespace_size 
--------------------
               4096
(1 row)

CREATE TABLE normal_table(id int) TABLESPACE normal_tablespace;
SELECT pg_tablespace_size('normal_tablespace');
 pg_tablespace_size 
--------------------
               8192
(1 row)

CREATE TABLESPACE compress_tablespace RELATIVE LOCATION 'compress_tablespace';
SELECT pg_tablespace_size('compress_tablespace');
 pg_tablespace_size 
--------------------
               4096
(1 row)

CREATE TABLE compressed_table_1024(id int) WITH(compresstype=2, compress_chunk_size=1024) TABLESPACE compress_tablespace;
SELECT pg_tablespace_size('compress_tablespace');
 pg_tablespace_size 
--------------------
               8192
(1 row)

DROP TABLE normal_table;
DROP TABLESPACE normal_tablespace;
DROP TABLE compressed_table_1024;
DROP TABLESPACE compress_tablespace;
